src/app/tests/TestCommandInteraction.cpp

```diff
@@ -1691,6 +1745,85 @@ void TestCommandInteraction::TestCommandHandlerAcceptMultipleCommands(nlTestSuit
     exchange->Close();
 }
 
+void TestCommandInteraction::TestCommandHandlerFillUpInvokeResponseMessageWhereSecondResponseIsStatusResponse(nlTestSuite * apSuite,
```

Comment:
Suggest `_` between TestCommandHandler and `FillUpXxxxx` for readability, in all the enormously long method names.